tests/system/src/test/java/test/robot/javafx/scene/NodeInitializationStressTest.java

```diff
@@ -182,26 +181,28 @@ public void areaChart() {
             c.getData().setAll(createNumberSeries());
             return c;
         }, (c) -> {
-            c.getData().setAll(createNumberSeries());
             accessChart(c);
+            c.getData().setAll(createNumberSeries());
+            c.getData().setAll(createNumberSeries());
```

Comment:
Just curious - why do we do setAll twice in all the tests?

Reply:
added a class-level comment.